src/sidebar/components/group-list-item-out-of-scope.js

```diff
@@ -1,37 +1,106 @@
+const { useState } = require('preact/hooks');
+const propTypes = require('prop-types');
+
 const {
   orgName,
   trackViewGroupActivity,
 } = require('../util/group-list-item-common');
 
-// @ngInject
-function GroupListItemOutOfScopeController(analytics) {
-  // Track whether the group details are expanded.
-  this.isDetailsExpanded = false;
+function GroupListItemOutOfScope({ analytics, group }) {
+  const [isExpanded, setExpanded] = useState(false);
 
-  /**
-   * Toggle the expanded setting on un-selectable groups.
-   */
-  this.toggleGroupDetails = function(event) {
+  const toggleGroupDetails = event => {
     event.stopPropagation();
-    this.isDetailsExpanded = !this.isDetailsExpanded;
-  };
-
-  this.orgName = function() {
-    return orgName(this.group);
+    setExpanded(!isExpanded);
   };
 
-  this.trackViewGroupActivity = function() {
+  const groupOrgName = orgName(group);
+  const trackViewActivity = event => {
+    event.stopPropagation();
     trackViewGroupActivity(analytics);
   };
+
+  return (
+    <div
+      className="group-list-item__item group-list-item-out-of-scope__item"
+      onClick={toggleGroupDetails}
+      tabIndex="0"
+    >
+      {/* the group icon */}
+      <div className="group-list-item__icon-container">
+        {group.logo && (
+          <img
+            className="group-list-item__icon group-list-item__icon--organization"
+            alt={groupOrgName}
+            src={group.logo}
+          />
+        )}
+      </div>
+      {/* the group name */}
+      <div
+        className={classnames({
+          'group-list-item-out-of-scope__details': true,
+          expanded: isExpanded,
+        })}
+      >
+        <svg
+          className="svg-icon group-list-item-out-of-scope__icon--unavailable"
+          xmlns="http://www.w3.org/2000/svg"
+          width="100%"
+          height="100%"
+          viewBox="0 0 24 24"
+        >
+          <path fill="none" d="M0 0h24v24H0V0z" />
+          <path d="M11 15h2v2h-2zm0-8h2v6h-2zm.99-5C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zM12 20c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z" />
+        </svg>
+        <a
+          className="group-list-item__name-link"
+          href=""
+          title="Group not annotatable on this domain."
+        >
+          {group.name}{' '}
+        </a>
+        <br />
+        {/* explanation of why group is not available */}
+        {!isExpanded && (
+          <p className="group-list-item-out-of-scope__details-toggle">
+            Why is this group unavailable?
+          </p>
+        )}
+        {isExpanded && (
+          <Fragment>
+            <p className="group-list-item-out-of-scope__details-unavailable-message">
+              This group has been restricted to selected URLs by its
+              administrators.
+            </p>
+            <p className="group-list-item-out-of-scope__details-actions">
+              <a
```

Comment:
Looks like we need to add some conditional logic here to prevent this link from showing up for third-party groups, as most restricted/open groups are third-party at this time, and there is no activity page to go to (yet).

Reply:
Done. This link is no longer rendered if the group has no HTML link.